politeiawww/api/v1/v1.go

```diff
@@ -147,6 +150,15 @@ const (
 	PropVoteStatusStarted     PropVoteStatusT = 2 // Proposal vote has been started
 	PropVoteStatusFinished    PropVoteStatusT = 3 // Proposal vote has been finished
 	PropVoteStatusDoesntExist PropVoteStatusT = 4 // Proposal doesn't exist
+
+	// Notification indentifiers
+	NotTypeSignupPaywallPaymentConfirmed NotificationType = 0
```

Comment:
I would prefer to prefix these with `Notification` instead of `NotType`, since `NotType` is a little confusing out of context. Also, can you rename the type to `NotificationT` to be consistent with the others?